signus/src/mainmenu.cpp

```diff
@@ -122,25 +125,44 @@ static int ProcessMenu(const char *mask1, const char *mask2) {
 			}
 
 			if (sel != oldsel) {
-				PlaySample(MenuSnd, 8, 32, 128);
-				DrawMN(pt1, pt2, bg, sel);
+				action = 1; // Select
 			}
-		}
-
-		if (e.What == evMouseDown) {
-			Flash(80, 250 + 77 * sel);
-			break;
-		}
-
-		if ((e.What == evKeyDown) && (e.Key.KeyCode == kbEsc)) {
+		} else if (e.What == evMouseDown) {
+			action = 2; // Flash
+		} else if (e.What == evKeyDown) {
+			switch (e.Key.KeyCode) {
+				// Flash
+				case kbEnter:
```

Comment:
Please keep `case` statements on the same indent level as the parent `switch`. Indent only the code in each `case`.

Reply:
Done!